src/shared/Core/Trace2FileWriter.cs

```diff
@@ -0,0 +1,18 @@
+using System.IO;
+
+namespace GitCredentialManager;
+
+public class Trace2FileWriter : Trace2Writer
+{
+    private readonly string _path;
+
+    public Trace2FileWriter(Trace2FormatTarget formatTarget, string path) : base(formatTarget)
+    {
+        _path = path;
+    }
+
+    public override void Write(Trace2Message message)
+    {
+        File.AppendAllText(_path, $"{Format(message)}\n");
+    }
+}
```

Comment:
do you folks care about final LFs on the source files?

Reply:
Er, we're not as strict as Git, but we prefer it. Thanks for pointing it out, I've got it fixed locally!